keyboards/redragon/k630/info.json

```diff
@@ -0,0 +1,190 @@
+{
+    "keyboard_name": "K630",
+    "manufacturer": "Redragon",
+    "url": "https://www.redragonzone.com/products/redragon-k630-gaming-mechanical-keyboard",
+    "maintainer": "bausensi",
+    "bootloader": "sn32-dfu",
+    
+    "processor": "SN32F248BF",
+    "usb": {
+        "vid": "0x0C45",
+        "pid": "0x5004",
+        "device_version": "0.0.1",
+        "force_nkro": true
+    },
+
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+
+    "debounce": 3,
```

Comment:
depending on your switches, you might be able to go as low as `1` here without compromising stability. A matter of preference I guess

Reply:
Understood. Making the changes right now.